vm/src/obj/objlist.rs

```diff
@@ -203,7 +203,7 @@ impl PyListRef {
                 return Ok(index);
             }
         }
-        let ref needle_str = vm.to_str(&needle)?.value;
+        let needle_str = vm.to_pystr(&needle)?;
```

Comment:
Was removing the `.value` intentional here? Will the `Display` impl for `PyString{,Ref}` do the right thing here?

Reply:
Oh, looking back on it it's probably not the best thing to do; thanks for pointing it out.